Controllers/AuthenticationController.js

```diff
@@ -21,6 +21,11 @@ function genToken(user) {
 export default {
   login: (req, res) => {
     User.findOne({ username: req.body.username }, (err, user) => {
+      if (!user) {
```

Comment:
This is not a good thing! You never want to explain that user was not found during authentication.

The credentials are either valid or invalid, if the user is not found it should return invalid credentials.... and that should be a separate PR?

Reply:
Ah I see apologies for that and thank you for clarifying for me!.

I was just fixing an unhandled error because in the next section there is an undefined access to the user object.

I will also do your suggestion and move this fix to another PR and returning an invalid credentials instead of not found.